src/components/SwiperFlatList/index.js

```diff
@@ -140,7 +140,7 @@ export default class SwiperFlatList extends PureComponent {
     this.setState({ paginationIndex: index });
 
     if (onMomentumScrollEnd) {
-      onMomentumScrollEnd();
+      onMomentumScrollEnd(index);
```

Comment:
Lets' send an object as a first parameter and the second `e` that has more data (nativeEvent, etc). Useful to not break the API in future changes.

```
onMomentumScrollEnd({ index }, e);
```

Reply:
i like it, ok